modules/util/create.py

```diff
@@ -794,6 +796,50 @@ def create_lr_scheduler(
                 optimizer,
                 initial_lr=optimizer.state_dict()['param_groups'][0]['initial_lr'],
             )
+        case LearningRateScheduler.CUSTOM:
+            # Special case. Unlike the others, we return from here.
+            if not config.custom_scheduler:
+                raise AssertionError("Must specify a class when using a custom LR scheduler.")
+            if "." not in config.custom_scheduler:
+                raise AssertionError("Custom class name must be in the format <module>.<class>")
+            klass = config.custom_scheduler.split(".")[-1]
+            module = config.custom_scheduler.removesuffix("." + klass)
+            module = importlib.import_module(module)
+            klass = getattr(module, klass)
+            # Compile arguments into single dict.
+            args = {}
+            for pd in config.scheduler_params:
+                key = pd["key"]
+                value = pd["value"]
+                # Special values
+                match value:
+                    case "%LR%":
```

Comment:
These special names should be documented somewhere. Ideally somewhere in the UI, close to the input fields.

Reply:
They are documented in the tooltips for the input fields.